src/Command/Playercount.js

```diff
@@ -69,7 +30,8 @@ module.exports = Command.extend({
             let key = this.getGameKey(tokens.join(' ').trim(), message.channel.id);
 
             if (key === 'help') {
-                let list = Object.values(gameNames)
+                let list = Object.keys(this.config.games.names)
+                    .map(key => this.config.games.names[key])
```

Comment:
Your change here will output a list of game names, instead of a markdown quote version of their ident